kfs-cam/src/main/resources/edu/arizona/kfs/module/cam/ojb-cam.xml

```diff
@@ -0,0 +1,57 @@
+<descriptor-repository version="1.0">
```

Comment:
We need to discuss this via Hangouts or phone as I don't understand why you would use OJB to resolve this issue. The wrong tool for the job.